index.html

```diff
@@ -1,18 +1,260 @@
+      </article>
+    </section>
+
+    <section class="skills">
+      <h2 class="skills__title">Skills</h2>
+      <ol class="skills__box">
+        <li class="skills__item">
+          <img src="img/skills/ui.svg" alt="ikona telefonu z pędzlem i dymkami chatu" />
+          <h4 class="skills__name">UX/UI</h4>
+          <h5 class="skills__secondName">Figma</h5>
+        </li>
+        <li class="skills__item">
+          <img class="skills__img" src="img/skills/icon_frontend.svg" alt="ikona laptopa z otwartymi dwoma szkicami stron internetowych" />
+          <h4 class="skills__name">Front End</h4>
+          <h5 class="skills__secondName">Html, CSS, JavaScript</h5>
+        </li>
+        <li class="skills__item">
+          <img class="skills__img" src="img/skills/icon_backend.svg" alt="ikona monitora wyświetlającego sześcian" />
+          <h4 class="skills__name">Back End</h4>
+          <h5 class="skills__secondName">PHP, Node.js</h5>
+        </li>
+      </ol>
+    </section>
+
+    <section class="portfolio">
+      <h2 class="portfolio__title">Portfolio</h2>
+      <!-- <p class="portfolio__hoverInfo">Uzyj Hover</p> -->
+      <div class="portfolio__box">
+        <figure class="portfolio__item portfolio__item--one">
+          <div class="portfolio__textContainer">
+            <h4 class="portfolio__name">Design</h4>
+            <p class="portfolio__desc">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            <button class="portfolio__btn btn btn--small">View</button>
+          </div>
+
+          <img class="portfolio__img" src="img/projects/project-1.png" alt="mapa na stole" />
+          <div class="portfolio__imgFiltr"></div>
+        </figure>
+
+        <figure class="portfolio__item portfolio__item--two">
+          <img class="portfolio__img" src="img/projects/project-2.png" alt="ładne rośliny stojącej na półce wiszącej na ścianie" />
+        </figure>
+
+        <figure class="portfolio__item portfolio__item--three">
+          <img class="portfolio__img" src="img/projects/project-3.png" alt="talerz z ładną potrawą" />
+        </figure>
+
+        <figure class="portfolio__item portfolio__item--four">
+          <img class="portfolio__img" src="img/projects/project-4.png" alt="fotel okok stoącej półki z kwiatkiem" />
+        </figure>
+
+        <figure class="portfolio__item portfolio__item--five">
+          <div class="portfolio__textContainer">
+            <h4 class="portfolio__name">Design project</h4>
+            <p class="portfolio__desc">xcepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```

Comment:
desc a description nie robi takiej wielkiej różnicy. Ogólnie w programowaniu lepiej stosować nazwy dłuższe niż skróty - tak żeby było jasne, o co chodzi. Oczywiście w tym przypadku jest to dość proste i ujdzie. No, ale chciałeś jakieś review no to masz :)

Reply:
zwrócę na to uwagę. Zawsze się nad tym zastanawiałem co lepiej użyć.
Kiedyś mówiono mi żeby stosować skróty jak "Btn" niż "button" bo jest to czytelniejsze przy długich nazwach.
Ale w takim razie będę o tym pamiętał w przypadku "desc"